go.mod

```diff
@@ -4,7 +4,7 @@ go 1.12
 
 require (
 	github.com/containernetworking/cni v0.8.1
-	github.com/containers/common v0.33.1
+	github.com/containers/common v0.34.0
```

Comment:
@rhatdan are you good with this bump?

Reply:
We will soon go to the latest.  This is fine for now.